packages/caliper-fabric/test/identity-management/IdentityManager.js

```diff
@@ -386,4 +405,56 @@ describe('An Identity Manager', () => {
         const identityManager = await identityManagerFactory.create(stubWalletFacadeFactory, [org1MSP, org2MSP]);
         await identityManager.getWallet().should.equal('IamAwallet');
     });
+
+    describe('when extracting identities from a specific wallet and store in the in memory wallet', () => {
+
+        it('if not identities available should not add anything to wallet', async () => {
+            const stubWalletFacadeFactory = sinon.createStubInstance(IWalletFacadeFactory);
+            const spyWalletFacade = sinon.createStubInstance(IWalletFacade);
+            stubWalletFacadeFactory.create.resolves(spyWalletFacade);
+            spyWalletFacade.getAllIdentityNames.resolves([]);
+            const identityManagerFactory = new IdentityManagerFactory();
+
+            const identityManager = await identityManagerFactory.create(stubWalletFacadeFactory, [org4MSP]);
+            await identityManager._extractIdentitiesFromWallet(org4MSP, spyWalletFacade);
```

Comment:
Unfortunately the `_` indicates a private method so the tests should not be calling that method directly but it should be driven through the use of appropriate external configurations.